### DISCUSSION TITLE
Creado el spring 1!!!!

### DISCUSSION THREAD

**Discussion by lionel1014** [pgCatamarca2021fs/aula1-grupoc-team-1c#22] [General]
Compañeres... ja... no digan compañeres... Hola a todos, he creado el spring 1 con una fecha estimada de cierre del 03/01/2022 (eeehh feliz año nuevo ¯\_( ͡° ͜ʖ ͡°)_/¯). El mismo estará centrada en la maquetación y en la configuración de los repositorios de cada uno. 
Con esto podemos empezar ya a codificar una landing page de presentación. 
Cualquier duda sin miedo a molestar a Ulises... jajaja no mentira Uli... sin miedo a consultarme ya vemos cuando nos juntamos a configurar el repositorio remoto de cada uno.
Saludos!!! 🐱‍👤

**Comment by lionel1014** [comment]
Además, me pareció útil que dejemos este formato de US#, así ya nos vamos familiarizando con leer la documentación... Cualquier cosa si les parece muy molesto no hay drama! lo puedo gestionar yo a eso.
![image](https://user-images.githubusercontent.com/77951627/147375191-028ac53d-bb84-4838-afdf-ae4855775d74.png)
Saludos!!! 🐱‍👤

**Comment by luisgm21** [comment]
No tendria que ser 03/01/2022

**Reply by lionel1014** [reply]
Tenes razón @luisgm21, igual verifique y el sprint, lo que puse mal fue el comentario... 
Saludos !!! 🐱‍👤

**Reply by luisgm21** [reply]
perfecto, si tambien corrobore pero ya quedaba medio pesado informarlo xD

**Reply by lionel1014** [reply]
No hay drama @luisgm21 !!! cualquier cosita que  te parezca raro y/o te llame la atención decilo... Total para eso esta la discusión! Sino te última _Duelo a muerte con cuchillos_ jajaj
